### PR TITLE
Updated Profiles

### PR DESCRIPTION
**Pull request by thatBrian**
**PR Type:** Update

**Related #'s:** N/A

Updating Profiles of Eboard Members.

## Description

Updated positions, meta_orders, and descriptions. Removed old members and added new members. 
## Motivation and Context

Required as new semester has arrived.

## Checklist:
- [x] My code follows the [code style of this project][code-style]
- [x] I have read the [**CONTRIBUTING** document][contributing]

[contributing]: https://github.com/BUGS-NYU/bugs-nyu.github.io/blob/master/.github/CONTRIBUTING.md
[code-style]: https://github.com/BUGS-NYU/bugs-nyu.github.io/tree/master/docs/STYLE_GUIDE.md

### CODE REVIEW

File: _contributors/knguyen.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Khanh Nguyen # Name of person
+position: Directory of Communications # Position, only displayed if in eboard
```
Comment (A1Liu) [inline _contributors/knguyen.md]: Can you change 'Directory' to 'Director' ?

File: _contributors/jzhang.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: Jiayue(Joanna) Zhang # Name of person
+position: Directory of Marketing # Position, only displayed if in eboard
```
Comment (A1Liu) [inline _contributors/jzhang.md]: Can you change 'Directory' to 'Director' ?

File: _contributors/aliu.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 name: Albert Liu # Name of person
-position: Webmaster # Position, only displayed if in eboard
+position: President # Position, only displayed if in eboard
 type: eboard # Can be one of [ eboard, alum, founder, mentor, member ]
-meta_order: 5
+meta_order: 10
 ---
 Hi I'm Albert.
```
Comment (A1Liu) [inline _contributors/aliu.md]: Can you change this to "Albert enjoys python scripting and long walks in Central Park. In his work time he majors in Computer Science at CAS."

Comment (A1Liu) [reply _contributors/aliu.md]: I changed my bio to "Hi I'm Albert" when I was refactoring a while ago and forgot to change it back.

File: _contributors/dtran.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 name: Deanna Tran
 position: Director of Communications
-type: eboard
+type: alum
```
Comment (A1Liu) [inline _contributors/dtran.md:3]: Could you do the same thing for Veena and Shelly?

File: _contributors/slin.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 name: Samuel Lin
 position: Vice Project Lead
```
Comment (A1Liu) [inline _contributors/slin.md]: Could you change this to Vice President?

**Review by A1Liu — CHANGES_REQUESTED** [review]
Looks good except for a few minor mistakes here and there.

**Comment by thatBrian** [comment]
Fixed, please check. @A1Liu

File: _contributors/aliu.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 name: Albert Liu # Name of person
-position: Webmaster # Position, only displayed if in eboard
+position: President # Position, only displayed if in eboard
 type: eboard # Can be one of [ eboard, alum, founder, mentor, member ]
-meta_order: 5
+meta_order: 10
 ---
-Hi I'm Albert.
+Can you change this to "Albert enjoys python scripting and long walks in Central Park. In his work time he majors in Computer Science at CAS."
```
Comment (A1Liu) [inline _contributors/aliu.md]: Please remove the "Can you change this" and the quotations.

**Review by A1Liu — CHANGES_REQUESTED** [review]
Just the one thing and I'll approve.

**Pull request merged**